src/config/firefox-desktop.js

```diff
@@ -46,6 +46,27 @@ export default {
       },
     },
   },
+  getParamsForQueryString(store) {
+    return {
+      channel: store.productDimensions.channel,
+      os: store.productDimensions.os,
+      aggregationLevel: store.productDimensions.aggregationLevel,
+      process: store.productDimensions.process,
+      timeHorizon: store.timeHorizon,
+      proportionMetricType: store.proportionMetricType,
+      activeBuckets: store.activeBuckets,
+      visiblePercentiles: store.visiblePercentiles,
+    };
+  },
+  getParamsforDataAPI(store) {
+    return {
+      channel: store.productDimensions.channel,
+      os: store.productDimensions.os,
+      probe: store.probeName,
+      process: store.productDimensions.process,
+      aggregationLevel: store.productDimensions.aggregationLevel,
```

Comment:
let's file an issue for this specifically. I think we're using `firefox` here which is probably not appropriate. We need to make sure we support the keys `firefoxDesktop` and `firefoxMobile` as two separate entities.

Reply:
The [constants.py](https://github.com/mozilla/glam/blob/main/glam/api/constants.py#L4-L7) file shows what I'm expecting in the API.

We can always change those keys. For mobile it may make sense to follow the app store IDs, e.g. `org.mozilla.fenix`.